Generals/Code/GameEngine/Source/GameNetwork/ConnectionManager.cpp

```diff
@@ -53,6 +53,45 @@
 #include "GameClient/DisconnectMenu.h"
 #include "GameClient/InGameUI.h"
 
+//============================================================================
+// hasValidTransferFileExtension
+//============================================================================
+// TheSuperHackers @security bobtista 06/11/2025
+// Validates file extensions for map transfer operations to prevent arbitrary
+// file types from being transferred over the network or loaded from save games.
```

Comment:
Can remove these comments here because down at callsite it already has sufficient comment.